woocommerce/woocommerce-ios
# [Analytics Hub] Initialize Revenue and Orders Analytics cards with localized placeholder data

Part of #8149 
Part of #8150 

## Description

This PR updates the default Revenue and Orders Analytics cards with localized placeholder data:

* Localizes the title strings on each card.
* Adds placeholders for the values and delta percentages.
* Adds a helper for calculating the delta tag color.

## Testing

1. Launch the app
2. Tap the "See More" button from the main dashboard
3. See the Analytics view loads with the placeholder data

## Screenshots

<img src="https://user-images.githubusercontent.com/8658164/203814044-ea414919-319e-439b-aff9-5ddd9c940603.png" width="300px">


## Submitter Checklist

Update release notes:
